packages/sources/react/src/index.ts

```diff
@@ -11,6 +11,7 @@ export { VtmnRadioButton } from './components/VtmnRadioButton';
 export { VtmnTextInput } from './components/VtmnTextInput';
 export { VtmnToggle } from './components/VtmnToggle';
 export { VtmnTooltip } from './components/VtmnTooltip';
+export { VtmnSelect } from './components/VtmnSelect';
```

Comment:
Can you please reorder alphabetically? Thanks!

Reply:
done